x-pack/plugins/security_solution/public/risk_score/containers/all/index.tsx

```diff
@@ -5,21 +5,10 @@
  * 2.0.
  */
 
-import deepEqual from 'fast-deep-equal';
-import { noop } from 'lodash/fp';
-import { useCallback, useEffect, useRef, useState } from 'react';
-import { Subscription } from 'rxjs';
+import { useEffect, useMemo } from 'react';
 
-import { isCompleteResponse, isErrorResponse } from '@kbn/data-plugin/common';
 import { createFilter } from '../../../common/containers/helpers';
-import { useKibana } from '../../../common/lib/kibana';
-import type {
-  RiskScoreStrategyResponse,
-  HostsRiskScore,
-  UsersRiskScore,
-  RiskScoreSortField,
-  RiskScoreRequestOptions,
-} from '../../../../common/search_strategy';
+import type { RiskScoreSortField, StrategyResponseType } from '../../../../common/search_strategy';
```

Comment:
Just a note that it might be worth breaking these apart into separate files. (i.e. `use_user_risk_score.tsx`, `use_host_risk_score.tsx` and they both import from `use_risk_score.tsx`. Not necessary for this PR, but maybe something to do in the future if there's time